app/Rules/MetaWikidataValue.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace App\Rules;
+
+use App\Services\WikibaseAPIClient;
+use Illuminate\Contracts\Validation\Rule;
+
+class MetaWikidataValue implements Rule
+{
+    /**
+     * @var WikibaseAPIClient Wikidata API Client
+     */
+    private $wikidata;
+
+
+    /**
+     * Create a new rule instance.
+     *
+     * @return void
+     */
+    public function __construct(WikibaseAPIClient $wikidata)
+    {
+        $this->wikidata = $wikidata;
+    }
+
+
+    /**
+     * Determine if the validation rule passes.
+     *
+     * @param  string  $attribute
+     * @param  mixed  $value
+     * @return bool
+     */
+    public function passes($attribute, $value)
+    {
+        [
+            'property' => $property,
+            'meta_value' => $metaWikidataValue
+        ] = $value;
+
+        try {
+            if (!empty($property) && !empty($metaWikidataValue)) {
+                $dataTypes = $this->wikidata->getPropertyDatatypes([$property]);
+                return array_key_exists($property, $dataTypes) && $dataTypes[$property] == 'time';
```

Comment:
IMHO this `array_key_exists()` shouldn’t be here – `getPropertyDatatypes()` should guarantee that the property is included in the return value. (It looks like it currently doesn’t do that, so I’ll upload a pull request to fix that.)

Reply:
Actually that's why I had to add this check. Ping me when you open a PR for that then I will update this part